### PR TITLE
Updated README.md and citations

### PR DESCRIPTION
**Pull request by fgrunewald**
Updated README file

- [x] improve functionality section
- [x] update final citation
- [x] add badge for tests 
- [x] add citations for GROMOS polymers
- [x] add citations for M3 polymers
- [x] add citation for polyply for all lib -files

### CODE REVIEW

File: polyply/data/martini3/citations.bib [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+author={Souza, Paulo C T and Alessandri, Riccardo and Barnoud, Jonathan and Thallmair, Sebastian and Faustino, Ignacio and Grünewald, Fabian and Patmanidis, Ilias and Abdizadeh, Haleh and Bruininks, Bart M H and Wassenaar, Tsjerk A and Kroon, Peter C and Melcr, Josef and Nieto, Vincent and Corradi, Valentina and Khan, Hanif M and Domański, Jan and Javanainen, Matti and Martinez-Seara, Hector and Reuter, Nathalie and Best, Robert B and Vattulainen, Ilpo and Monticelli, Luca and Periole, Xavier and Tieleman, D Peter and de Vries, Alex H and Marrink, Siewert J},
+doi={10.1038/s41592-021-01098-3},                                             
+issn={1548-7105},                                                             
+journal={Nature Methods},                                                     
+title={{Martini 3: a general purpose force field for coarse-grained molecular dynamics}},
```
Comment (ricalessandri) [inline polyply/data/martini3/citations.bib:7]: You could add:
```
volume={18},
pages={382--388},
```

**Review by ricalessandri — COMMENTED** [review]
Just a few small things!

File: README.md [hunk 2]
```diff
@@ -1,47 +1,60 @@
...
-@article{grunewald2021polyply,
-  title={Polyply: a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+@article{Grunewald2021Polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
   author={Gr{\"u}newald, Fabian and Alessandri, Riccardo and Kroon, Peter C and 
   	  Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
-  journal={arXiv preprint arXiv:2105.05890},
-  year={2021}
+  journal={Nature Communications},
+  volume={TBA},
```
Comment (ricalessandri) [inline README.md:53]: add: 
```
volume={13},
pages={68},
```

File: polyply/data/2016H66/citation.bib [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  pages={3825--3850},
+  year={2016},
+  publisher={ACS Publications}
+}
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
+  doi={10.1038/s41467-021-27627-4},
```
Comment (ricalessandri) [inline polyply/data/2016H66/citation.bib:16]: following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```

File: polyply/data/gromos53A6/citations.bib [hunk 4]
```diff
@@ -23,3 +23,12 @@ @article{P3HT_GROMOS
   doi      ={10.1021/jacs.6b11717},
   url      ={https://doi.org/10.1021/jacs.6b11717}
 }
+
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```
Comment (ricalessandri) [inline polyply/data/gromos53A6/citations.bib:31]: following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```

File: polyply/data/martini2/citations.bib [hunk 5]
```diff
@@ -70,3 +70,10 @@ @article{P3HT_M2
   doi={10.1021/jacs.6b11717}
 }
 
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```
Comment (ricalessandri) [inline polyply/data/martini2/citations.bib:76]: following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```

File: polyply/data/martini3/citations.bib [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+url={https://doi.org/10.1038/s41592-021-01098-3},                             
+year={2021}                                                                   
+}
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```
Comment (ricalessandri) [inline polyply/data/martini3/citations.bib:14]: following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```

File: polyply/data/martini3_beta/citations.bib [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  pages={7436--7449},
+  year={2018},
+  publisher={ACS Publications},
+  doi={10.1021/acs.jpcb.8b04760}
+}
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```
Comment (ricalessandri) [inline polyply/data/martini3_beta/citations.bib:16]: following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```

File: polyply/data/oplsaaLigParGen/citations.bib [hunk 8]
```diff
@@ -35,3 +35,11 @@ @article{OPLSLigParGen3
   year={2017},
   doi={10.1021/acs.jpcb.7b00272}
 }
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```
Comment (ricalessandri) [inline polyply/data/oplsaaLigParGen/citations.bib:42]: following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```

File: polyply/data/parmbsc1/citations.bib [hunk 9]
```diff
@@ -8,3 +8,11 @@ @article{parmbsc1
   year={2016},
   publisher={Nature Publishing Group}
 }
+
+@article{polyply,
+  title={Polyply; a python suite for facilitating simulations of (bio-) macromolecules and nanomaterials},
+  author={Grunewald, Fabian and Alessandri, Riccardo and Kroon, Peter C and Monticelli, Luca and Souza, Paulo CT and Marrink, Siewert J},
+  journal={Nature Communications},
```
Comment (ricalessandri) [inline polyply/data/parmbsc1/citations.bib:15]: following https://www.nature.com/articles/s41467-021-27627-4, you can already add: 
```
volume={13},
pages={68},
```

**Review by ricalessandri — CHANGES_REQUESTED** [review]
Add volume and article numbers (as "pages") for the polyply citation.

**Review by ricalessandri — APPROVED** [review]
Optional change still missing in main `README.md`:
`Grunewald2021Polyply` --> `Grunewald2022Polyply`, just in case you missed it

**Comment by fgrunewald** [comment]
@ricalessandri good call fixed it

**Review by ricalessandri — APPROVED** [review]
Great!

**Pull request merged**